scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -1017,14 +942,17 @@ def processPipelineJsonForExistingGates(cookie):
                                               payloadConstraint, pipelineId, env_json)
             elif gateType.__eq__("verification"):
                 stageJson = verificationGateProcess(applicationId, serviceId, gateId, gateName, gateType,
-                                                    payloadConstraint, pipelineId, env_json, cookie)
+                                                    payloadConstraint, pipelineId, env_json)
             elif gateType.__eq__("approval"):
                 stageJson = approvalGateProcess(applicationId, serviceId, gateId, gateName, gateType,
-                                                payloadConstraint, pipelineId, env_json, cookie)
+                                                payloadConstraint, pipelineId, env_json)
             if stageJson is not None:
                 stageJson["application"] = appName
                 logging.info(f"StageJson is :  {stageJson}")
                 pipelineJson = updatePipelineJson(pipelineId, stageJson)
+                cookie = "no-cookie"
+                if isd_commongate == "yes":
```

Comment:
@sudhakaropsmx input value "true / false" will be more suitable ?

Reply:
removed commongate enabled